applications/dashboard/controllers/class.entrycontroller.php

```diff
@@ -1269,27 +1269,41 @@ public function register($InvitationCode = '') {
 
         $this->setData('NoEmail', UserModel::noEmail());
 
-        $RegistrationMethod = $this->_registrationView();
-        $this->View = $RegistrationMethod;
-        $this->setData('Method', stringBeginsWith($RegistrationMethod, 'Register', false, true));
-        $this->$RegistrationMethod($InvitationCode);
+        // Sub-dispatch to a specific handler for each registration method
+        $registrationHandler = $this->getRegistrationhandler();
+        $this->setData('Method', stringBeginsWith($registrationHandler, 'register', true, true));
+        $this->$registrationHandler($InvitationCode);
     }
 
     /**
      * Select view/method to be used for registration (from config).
      *
      * @access protected
-     * @since 2.0.0
-     *
-     * @return string Method name.
+     * @since 2.2.106
```

Comment:
I've been going with the standard of labeling these with the first _release_ it will be since / deprecated in, which is 2.3 in this case. I think that makes better sense for folks outside the company.